colour/colorimetry/blackbody.py

```diff
@@ -594,3 +600,513 @@ def sd_blackbody(
         wavelengths,
         name=f"{temperature}K Blackbody",
     )
+
+
+def rayleigh_jeans_law(
+    wavelength,
+    temperature,
+) -> FloatingOrNDArray:
+    """
+    Returns the spectral radiance of electromagnetic radiation as a
+    function of wavelength from a black body at a given temperature
+    :math:`T[K]` in a medium having index of refraction :math:`n`.
```

Comment:
``index of refraction :math:`n`.`` --> ``index of refraction :math:`n` according to *Rayleigh–Jeans* law.``